tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -415,7 +415,8 @@ internal static IApi GetApi(TracerSettings settings, IDogStatsd statsd, Action<D
                         LanguageInterpreter = frameworkDescription.Name,
                         ComputeStats = settings.StatsComputationEnabled,
                         TelemetryClientConfiguration = telemetryClientConfiguration,
-                        ClientComputedStats = clientComputedStats
+                        ClientComputedStats = clientComputedStats,
+                        RatesPayloadVersion = false
```

Comment:
I've checked that this was included in the agent version 7.27.0 which was released 4 years ago. I don't know if it could be possible that any customer could be using such version, that's why I decided to disable it and cache the response as a fallback to maintain parity with the managed trace manager. In case we would need to support that scenario I think that we would need to handle it internally in the trace exporter since at this point we don't have the agent version. @andrewlock WDYT?